docs/source/en/api/pipelines/text_to_video_zero.mdx

```diff
@@ -174,14 +178,14 @@ To perform text-guided video editing (with [InstructPix2Pix](./stable_diffusion/
     ```
 
 
-### Dreambooth specialization 
+### DreamBooth specialization 
```

Comment:
For this one, we are not using `video` but instead, `pose_images`. I think this needs correction. Could you please advise?

Reply:
You mean the title of the step 1?

Reply:
No. In the [code example](https://huggingface.co/docs/diffusers/main/en/api/pipelines/text_to_video_zero#dreambooth-specialization), you initialized `video` but never used it. `pose_images` was used instead.